docs/buttons.stories.js

```diff
@@ -0,0 +1,18 @@
+import { storiesOf } from '@storybook/html'; // eslint-disable-line import/no-extraneous-dependencies
+import { // eslint-disable-line import/no-extraneous-dependencies
+  withKnobs, select,
+} from '@storybook/addon-knobs';
+
+const stories = storiesOf('Buttons', module);
+stories.addDecorator(withKnobs);
+
+stories.add('button', () => {
+  const extraClass = select('class', {
+    btn: 'btn',
+    'btn is-primary': 'btn is-primary',
+    'btn is-success': 'btn is-success',
+    'btn is-warning': 'btn is-warning',
+    'btn is-error': 'btn is-error',
+  }, 'btn');
```

Comment:
I think radios are easier to use. 👍 
```js
const extraClass = radios('class', {
      default: '',
      'is-primary': 'is-primary',
      'is-success': 'is-success',
      'is-warning': 'is-warning',
      'is-error': 'is-error',
  }, '');
```
![buttons](https://user-images.githubusercontent.com/5305599/49491755-c3e3be00-f898-11e8-89a3-967f2c8e6896.gif)